samples/subsys/nvs/src/main.c

```diff
@@ -154,6 +155,30 @@ void main(void)
 		}
 	}
 
+	/* LONG_ID is used to store a larger dataset ,lets see if we can read
+	 * it from flash
```

Comment:
note that "larger dataset" means those which are using the long format of length fields.

Reply:
Yes